cookbooks/bcpc-hadoop/attributes/hbase.rb

```diff
@@ -51,8 +51,8 @@
 default["bcpc"]["hadoop"]["hbase_rs"]["xmn"]["size"] = 256
 default["bcpc"]["hadoop"]["hbase_rs"]["xms"]["size"] = 1024
 default["bcpc"]["hadoop"]["hbase_rs"]["xmx"]["size"] = 1024
-default["bcpc"]["hadoop"]["hbase_rs"]["mx_dir_mem"]["size"] = 256
-default["bcpc"]["hadoop"]["hbase_rs"]["hdfs_dir_mem"]["size"] = 128
+default["bcpc"]["hadoop"]["hbase_rs"]["mx_dir_mem"]["size_by_ratio"] = 0.2 # of total memory
```

Comment:
The reason to use ratio is that worker nodes have different amount of memory. (For example, 128GB and 256GB). When the bucket cache is relatively small, say 16GB, it is fine to use fixed number.
However, a fixed size of 100 GB bucket cache probably cannot work on a 128GB worker node.